rosco-manifests/src/test/groovy/com/netflix/spinnaker/rosco/manifests/kustomize/KustomizationFileReaderSpec.groovy

```diff
@@ -43,24 +46,23 @@ class KustomizationFileReaderSpec extends Specification {
             .type("github/file")
             .build()
 
+        mockRetrofit.when(Retrofit2SyncCall.execute(clouddriverService.fetchArtifact(baseArtifact)))
```

Comment:
I'm still curious about this mocking setup.  I'd expect to see one level of mocking for `clouddriverService.fetchArtifact(baseArtifact)`, and then a second one `mockRetrofit.execute`.